e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/delete-payment.spec.js

```diff
@@ -64,16 +64,16 @@ context(`${PDC_TEAMS.PDC_RECONCILE} users can delete payments`, () => {
 
     pages.utilisationReportPage.premiumPaymentsTab.clickPaymentLink(payment.id);
 
-    cy.url().should('eq', relative(`/utilisation-reports/${report.id}/edit-payment/${payment.id}`));
+    cy.url().should('eq', relative(`/utilisation-reports/${report.id}/edit-payment/${payment.id}?redirectTab=premium-payments`));
```

Comment:
Could a const be made for `/utilisation-reports/${report.id}/edit-payment/${payment.id}`  is repeated